# Generated for NaoQi method ALSensors.run
# Monitors sensors.

---


